sos/plugins/sssd.py

```diff
@@ -29,6 +29,7 @@ def setup(self):
         self.add_copy_spec([
             "/etc/sssd/sssd.conf",
             "/var/log/sssd/*",
+            "/var/lib/sss/pubconf/krb5.include.d/*",
```

Comment:
No, that's fine - so long as someone familiar with the component doesn't normally expect secrets to be found there (and we can always try to blame Simo ;-) ).